packages/sdk/src/decode/decode.js

```diff
@@ -75,6 +58,21 @@ const decodeComposite = async (composite, decoders, stack) => {
   return decoder ? await decoder(decoded) : decoded
 }
 
+const decodeInclusiveRange = async (range, decoders, stack) => {
+  // Recursive decode for start, end, and step
+  // We don't do all fields just in case there are future API changes
+  // where fields added and are not Cadence values
+  const keys = ["start", "end", "step"]
+  const decoded = await Object.keys(range).reduce(async (acc, key) => {
+    acc = await acc
+    if (keys.includes(key)) {
+      acc[key] = await recurseDecode(range[key], decoders, [...stack, key])
```

Comment:
Do you care about handling if these fail?

Reply:
Nope, I think these should just propogate up (also this follows the logic of existing decoders)